### PR TITLE
DEVX-2070: Update default datagen version in Dockerfile-local to 6.0.0

### PR DESCRIPTION
**Pull request by rspurgeon**
Updating the default only, this has no material affect on the current demo behavior as it is set in script by start-docker.sh

### CODE REVIEW

File: ccloud/Dockerfile-local [hunk 1]
```diff
@@ -17,7 +17,7 @@ ARG CP_VERSION
 
 FROM confluentinc/cp-server-connect-base:$CP_VERSION
 
-ARG CONNECTOR_VERSION=5.5.0-SNAPSHOT
+ARG CONNECTOR_VERSION=6.0.0-SNAPSHOT
```
Comment (ybyzek) [inline ccloud/Dockerfile-local]: @rspurgeon should this be `6.0.0` instead?

Comment (rspurgeon) [reply ccloud/Dockerfile-local]: @ybyzek .. yes.  Thank you! 🤦

Comment (rspurgeon) [reply ccloud/Dockerfile-local]: @ybyzek did you see my correction for this ?

**Review by ybyzek — APPROVED** [review]
LGTM

**Pull request merged**